WEB_SERVICE/TLS/_MEDIUM/TLS_FREAK_ATTACK/meta.json

```
@@ -0,0 +1,29 @@
+{
+  "risk_rating": "medium",
+  "short_description": "Support for export-grade RSA encryption allowing downgrade attacks and cryptographic factoring of weak 512-bit keys.",
+  "references": {
+    "CVE-2015-0204": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-0204",
```

I suggest putting nvd since it has more details
```suggestion
    "CVE-2015-0204": "https://nvd.nist.gov/vuln/detail/CVE-2015-0204",
```